src/core/services/compiler/compiler.js

```diff
@@ -112,12 +112,19 @@ function mdCompilerService($q, $templateRequest, $injector, $compile, $controlle
         link: function link(scope) {
           locals.$scope = scope;
 
-          //Instantiate controller if it exists, because we have scope
+          // Instantiate controller if it exists, because we have scope
           if (controller) {
-            var invokeCtrl = $controller(controller, locals, true, controllerAs);
+
+            var ctrlLocals = angular.extend(locals, {
+              $element: element
```

Comment:
It's still not possible. Though `content` elements are still not dynamic, those are just to avoid compilation each time and are `pre-rendered` elements.